.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea/
+/pdfSourceJRD.html
```

Comment:
what are these files?

Reply:
.idea is generated by Writerside  
pdfSource* and webHelpJRD2-all.zip are the local version that are generated when doing local tests